AUTHORS

```diff
@@ -63,6 +63,7 @@ Laurent Raufaste
 Jean Remond
 Andrew Stribblehill
 James Thompson
+Aleksey Tsalolikhin
```

Comment:
Are you sure you have less than 10 commits?

Reply:
@vpodzime I've only made one commit to the C code  :)

@olehermanse thanks, I've added a few email accounts -- I'll verify them later today. I appreciate the tip!  :)